client/app/components/composites/PageSelection/PageSelection.js

```diff
@@ -0,0 +1,43 @@
+import r, { div, a } from 'r-dom';
+import { ArrowButton } from '../../elements/RoundButton/RoundButton';
+import { upsertSearchQueryParam } from '../../../utils/url';
+
+import css from './PageSelection.css';
+
+export default function PageSelection({ currentPage, totalPages, location, pageParam }) {
+  const BUTTON_SIZE = '36px';
```

Comment:
Alternatively size could be handled in CSS, and this component could just add className to get the correct effect. (That would make responsive layouts a bit easier.)